esp/src/src/react/wuStatus.tsx

```diff
@@ -25,7 +23,7 @@ const Steps = [
     },
     {
         text: nlsHPCC.Completed,
-        activeText: nlsHPCC.Completing
+        activeText: nlsHPCC.Completed
```

Comment:
@miguelvazq @kunal - this was caught at compile time...  Plus you get autocompletion in TS files (which is v. useful).

Reply:
@GordonSmith I think you want to tag @kunalaswani